### DISCUSSION TITLE
Tutorial: Build Modern Cross-Platform Apps with .NET

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#8679] [General]
Half of the tutorial is already written, the rest will follow shortly.
https://github.com/mysteryx93/Modern.Net-Tutorial

I hope it will help some people sharpen their skills!

**Comment by timunie** [comment]
Nice read. Consider adding it AwsaomeAvalonia 🙂

**Reply by mysteryx93** [reply]
I created a Pull Request in AwesomeAvalonia
https://github.com/AvaloniaCommunity/awesome-avalonia/pull/77

**Comment by mysteryx93** [comment]
Tutorial is completed!

Contributions are welcomed for the Deployment page to add Inno Setup for Windows builds, and automated builds, if someone goes down that road.

**Reply by mysteryx93** [reply]
Added to the AUR section; since I found myself digging in the documentation to find the commands up update a Linux Arch AUR package every time. This will save me time.
https://github.com/mysteryx93/Modern.Net-Tutorial/blob/main/8_Deployment.md#linux-arch-aur
